### PR TITLE
Removed extra imports, ordered methods in the services and went throu…

### PR DESCRIPTION
**Pull request by rhacp**
Removed extra imports from all classes, ordered methods in the services and went through all SonalLint suggestions.

### CODE REVIEW

File: src/main/java/com/teamrocket/tms/services/task/TaskServiceValidationImpl.java [hunk 1]
```diff
@@ -28,7 +28,7 @@ public void validateTaskAlreadyExists(TaskDTO taskDTO) {
 
     @Override
     public void validateTaskCanBeAssigned(Task task) {
-        if(task.isComplete() == true || task.getUser() != null){
+        if(task.isComplete() || task.getUser() != null){
```
Comment (AlexIliescu11) [inline src/main/java/com/teamrocket/tms/services/task/TaskServiceValidationImpl.java]: Please don't forget to format the code. Between `){` should be a space, that's a sign that the class wasn't formatted. 
Please consider formatting each class updated.

**Comment by AlexIliescu11** [comment]
Great job!

File: src/main/java/com/teamrocket/tms/services/task/TaskServiceValidationImpl.java (hunk 1 above)
Comment (rhacp) [reply src/main/java/com/teamrocket/tms/services/task/TaskServiceValidationImpl.java]: Thanks for the review! Just formatted all classes and went through the changes.

**Review by Daniel7Ionescu — APPROVED** [review]
looks good bro

**Pull request merged**